src/OrbitQt/HistogramWidget.cpp

```diff
@@ -136,7 +143,7 @@ static void DrawHistogram(QPainter& painter, const QPoint& axes_intersection,
               ValueToAxisLocation(bin_to, horizontal_axis_length, min_value, histogram.max),
           axes_intersection.y() - vertical_shift);
       const QRect bar(top_left, lower_right);
-      painter.fillRect(bar, Qt::cyan);
+      painter.fillRect(bar, kBarColors[i % kBarColorsCount]);
```

Comment:
Maybe `kBarColors.size()` instead of `kBarColorsCount`?

Reply:
Do you mean, it's easier to understand, that out-of-bound is not going to happen?

Reply:
Less error prone, when you add a new color you don't need to update that constant.

Reply:
Yes. It's also simpler to understand when you just see this line and haven't see the definition. kBarColorsCount could potentially be smaller than kBarColors.size(). It just conveys slightly more meaning while having the same advantages. But it's rather a nit. If you prefer your way, that's also okay for me!

Reply:
Yes. My assumption was you don't know the type. You only know it has operator[], so it's some kind of list. But as I said before, it's rather a Nit.